script.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+	//DO NOT CHANGE CODE BELOW
+        var num1 = parseInt($('#num1').val());
+        var num2 = parseInt($('#num2').val());
+        clear();
+        addNumbers(num1, num2);
+        var result = multiplyNumbers(num1, num2);
+        $('#result-mult').text(result);
+	//DO NOT CHANGE CODE ABOVE
+
+    });
+
+	/*
+	 Below this comment, create a function 
+	 named addNumbers, which accepts two parameters.
+	 The function should add the two parameters together
+	 and write the result to the element with the id
+	 result-add
+	*/
+function addNumbers(num1, num2) {
+    var result = num1 + num2;
+    return result};

```

Comment:
when you return in a function - nothing else happens after. So - line 24 will never execute - meaning that this function will not set the value of #result-add as is required. I would delete line 24.